### PR TITLE
Fix markers URL in the new applications

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: build.mk [hunk 1]
```diff
@@ -125,6 +125,7 @@ $(APPS_PACKAGE_PATH)/static/apihelp: /usr/lib/node_modules/ngeo/api/dist/apihelp
 	sed -i -e 's#"data\.txt"#"../static/$${CACHE_VERSION}/apihelp/data.txt"#g' $@/index.html.tmpl_tmpl
 	sed -i -e "s#'data\.txt'#'../static/\$${CACHE_VERSION}/apihelp/data.txt'#g" $@/index.html.tmpl_tmpl
 	sed -i -e 's#img/#../static/$${CACHE_VERSION}/apihelp/img/#g' $@/index.html.tmpl_tmpl
+	sed -i -e "s#https://geomapfish-demo-2-[0-9].camptocamp.com/0/img/markers/#../static/0/apihelp/markers/#g" $@/data.txt
```
Comment (ochriste) [inline build.mk:128]: why is there a reference to the demo in the c2cgeoportal build code ?

Comment (ger-benjamin) [reply build.mk:128]: Previously, these markers was on the openlayers 2 example server (yes, the v2).
We have moved them on a server we own. This one was the better we have found.

**Review by ger-benjamin — APPROVED** [review]

File: build.mk (hunk 1 above)
Comment (sbrunner) [reply build.mk:128]: To get the apps and API help from ngeo...

Comment (ochriste) [reply build.mk:128]: @ger-benjamin mmm so it's like an external online resource?
@sbrunner Im sorry, I dont understand your answer, but you will be able to explain to me more clearly during the daily monday

**Pull request merged**